From 33813d72a8651b578b9b13d562f723f99c6d75eb Mon Sep 17 00:00:00 2001
From: Ada Tufankjian <amt3824@g.rit.edu>
Date: Fri, 6 Sep 2024 16:29:07 -0700
Subject: [PATCH] Fixing build on Ubuntu 24.04 w/ Clang 20 (#936)

Some compiler implementations seem to be very adamant that
`std::unique_ptr` lives in `<memory>` and without this change I was
unable to build.
---
 tools/imageio/imageio.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/imageio/imageio.h b/tools/imageio/imageio.h
index 02e6ab5e38..83c1331b32 100644
--- a/tools/imageio/imageio.h
+++ b/tools/imageio/imageio.h
@@ -37,6 +37,7 @@
 #include <string>
 #include <sstream>
 #include <vector>
+#include <memory>
 
 #include <fmt/format.h>
 #include <math.h>
